src/clients/Templates.ts

```diff
@@ -0,0 +1,52 @@
+import { IOContext } from '@vtex/api'
+import { AxiosRequestConfig } from 'axios'
+import { HttpClient } from './HttpClient'
+
+const createRoutes = ({ account, workspace }: IOContext) => {
+  const routes = {
+    Templates: (argWorkspace: string) =>
+      `/${account}/${argWorkspace || workspace}/buckets/vtex.pages-graphql/userData/files/store/templates.json`,
```

Comment:
If the user doesn't pass workspace and he is @master workspace, he will put a file into the master workspace, am I right?

Reply:
Yup, you're right. Actually, the user is going to edit an existing file, but yes, they will migrate de master workspace